edm4eic.yaml

```diff
@@ -372,39 +391,40 @@ datatypes:
+      - uint64_t             surface           // Surface for bound parameters (geometryID)
+      - edm4hep::Vector2f    loc               // 2D location on surface
+      - float                theta             // Track polar angle [rad]
+      - float                phi               // Track azimuthal angle [rad]
+      - float                qOverP            // [e/GeV]
+      - float                time              // Track time [ns] 
+      - edm4eic::Cov6f       covariance        // Full covariance in basis [l0,l1,theta,phi,q/p,t]
+  
     OneToOneRelations:
       - edm4eic::Trajectory trajectory      // Trajectory associated with these track parameters
 
+
   edm4eic::Track:
     Description: "Track information at the vertex"
-    Author: "S. Joosten"
+    Author: "S. Joosten, J. Osborn"
     Members:
-      - int32_t           type              // Flag that defines the type of track
-      - float             chi2              // Total chi2 (sum) of the track fit
-      - int32_t           ndf               // Numbers of degrees of freedom of the track fit
-      - edm4hep::Vector3f momentum          // Track 3-momentum at the vertex [GeV]
-      - edm4eic::Cov3f    momentumError     // Covariance matrix on the momentum
-      - float             time              // Track time at the vertex [ns]
-      - float             timeError         // Error on the track vertex time
-      - float             charge            // Particle charge
+      - int32_t            type                           // Flag that defines the type of track
+      - edm4hep::Vector3f  position                       // Track 3-position at the vertex 
+      - edm4hep::Vector3f  momentum                       // Track 3-momentum at the vertex [GeV]
+      - edm4eic::Cov6f     positionMomentumCovariance     // Covariance matrix in basis [x,y,z,px,py,pz]
+      - float              time                           // Track time at the vertex [ns]
+      - float              timeError                      // Error on the track vertex time
+      - float              charge                         // Particle charge
+      - float              chi2                           // Total chi2
+      - uint32_t           ndf                            // Number of degrees of freedom
+      - uint32_t           pid                            // PDG particle ID hypothesis
     OneToOneRelations:
-      - edm4eic::Trajectory trajectory      // Trajectory of this track
-      - edm4eic::Vertex   vertex            // Track vertex of this track
+      - edm4eic::Trajectory                     trajectory      // Trajectory of this track
     OneToManyRelations:
-      - edm4eic::TrackerHit trackerHits     // Hits that were used for this track
-      - edm4eic::Track    tracks            // Tracks (segments) that have been combined to create this track
+      - edm4eic::TrackerHit trackerHits       // Hits that were used for this track
```

Comment:
These should be `Measurement2D` now I think.

Reply:
Sure, I didn't know if we were ready to move to this

Reply:
Are we using Measurement2D now? Else we should first go to `TrackerHit` to decouple both PRs (but make a note of what it should be, and already add the Measurement2D version commented out).

Reply:
I think we are using `TrackerHit` so if we haven't swapped to `Measurement2D` yet then I agree we should decouple these changes.

Reply:
We are already using Measurement2D, https://github.com/eic/EDM4eic/pull/54. Maybe this branch needs a rebase?

Reply:
Is Measurement2D used anywhere in EICrecon though? I don't see it

Reply:
It's going to be merged today probably, https://github.com/eic/EICrecon/pull/1076

What matters is what is in this repo for this discussion.